app/views/articles/show.html.erb

```diff
@@ -66,9 +66,7 @@
     </script>
   <% end %>
 
-  <% if @article.processed_html.include? "runkit-element" %>
-    <%= javascript_include_tag "https://embed.runkit.com" %>
-  <% end %>
+  <%= javascript_include_tag "https://embed.runkit.com" %>
```

Comment:
is it possible to keep the if or with the new structure it isn't?

Reply:
With the `if` present, **preview of the single new Runkit comment won't work**. The reason for that is probably that the initial show page without Runkit comments won't have Runkit lib, and the lib won't get loaded, when the "PREVIEW" button is clicked.

If the Runkit lib was loaded on demand, when clicking "PREVIEW", it'd be possible to keep the `if`, but I feel it'd get too complicated, with 2 places handling that.